packages/@markuplint/file-resolver/src/ml-file.ts

```diff
@@ -20,7 +20,7 @@ export class MLFile {
 	constructor(filePathOrContext: string, anonymous = false, workspace = process.cwd(), name = '<AnonymousFile>') {
 		this.anonymous = anonymous;
 		if (anonymous) {
-			this.#filePath = `${workspace}/${name}`;
+			this.#filePath = path.resolve(workspace, name);
```

Comment:
I changed this line because:

1. It will resolve invalid path on Windows
2. it does not support absolute `name` path like my test case, it is unintuitive

Reply:
By the way, it seems `anonymous` actually means `virtual` here, a filename could be provided, then it's just could be virtual but not anonymous. I thought `anonymous` meant `<AnonymousFile>` at my first look.